app/components/UserProfile/PrimaryProfile.tsx

```diff
@@ -0,0 +1,36 @@
+import { Avatar } from "@mui/material";
+import React, { FC } from "react";
+
+type PrimaryProfileProps = {
+  name: string;
+  role: string;
+  avatarSrc: string;
+  className?: string;
+};
+
+const PrimaryProfile: FC<PrimaryProfileProps> = ({
+  name,
+  role,
+  avatarSrc,
+  className = "",
+}) => {
+  return (
+    <div className={className}>
+      <Avatar
+        className=""
+        alt={name}
+        src={avatarSrc}
+        sx={{
+          height: 150,
+          width: 150,
+        }}
+      />
+      <div className="flex flex-col items-center gap-2">
```

Comment:
```suggestion
      <div className="text-center">
```
Wrap the content inside a flex flex-col items-center container.
```
<div className={`flex flex-col items-center gap-4 ${className}`}>
      <Avatar
```

Reply:
the parent div is controlled by the coder. using like suggestion it's break project design.